src/controller/python/chip/FabricAdmin.py

```diff
@@ -152,14 +140,14 @@ def __init__(self, vendorId: int, rcac: bytes = None, icac: bytes = None, fabric
         try:
             adminList = builtins.chipStack.GetStorageManager().GetReplKey('fabricAdmins')
         except KeyError:
-            adminList = {str(self._fabricIndex): {'fabricId': self._fabricId}}
+            adminList = {str(self._adminIndex): {'fabricId': self._fabricId}}
             builtins.chipStack.GetStorageManager().SetReplKey('fabricAdmins', adminList)
 
-        adminList[str(self._fabricIndex)] = {'fabricId': self._fabricId, 'vendorId': self.vendorId}
+        adminList[str(self._adminIndex)] = {'fabricId': self._fabricId, 'vendorId': self.vendorId}
```

Comment:
If one AdminIndex can underpin N fabrics, why is there a fabricID here?

This seems odd

Reply:
No, one AdminID underpins one fabric. The alterations we discussed were to tease out the CA as a separate entity in a future change.

Reply:
See description of class.